QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -685,16 +814,69 @@ ElasticProblem<dim>::ElasticProblem()
   Y_AlGaAs[2][1][1][2] = c44;
   Y_AlGaAs[2][1][2][1] = c44;
 
+  Y_AlGaAs_Phi[0][0][0][0] = c11 * phi11;
+  Y_AlGaAs_Phi[1][1][1][1] = c11 * phi11;
+  Y_AlGaAs_Phi[2][2][2][2] = c11 * phi11;
+
+  Y_AlGaAs_Phi[0][0][1][1] = c12 * phi12;
+  Y_AlGaAs_Phi[0][0][2][2] = c12 * phi12;
+  Y_AlGaAs_Phi[1][1][0][0] = c12 * phi12;
+  Y_AlGaAs_Phi[1][1][2][2] = c12 * phi12;
+  Y_AlGaAs_Phi[2][2][0][0] = c12 * phi12;
+  Y_AlGaAs_Phi[2][2][1][1] = c12 * phi12;
+
+  Y_AlGaAs_Phi[0][1][0][1] = c44 * phi44;
+  Y_AlGaAs_Phi[0][1][1][0] = c44 * phi44;
+  Y_AlGaAs_Phi[1][0][0][1] = c44 * phi44;
+  Y_AlGaAs_Phi[1][0][1][0] = c44 * phi44;
+
+  Y_AlGaAs_Phi[0][2][0][2] = c44 * phi44;
+  Y_AlGaAs_Phi[0][2][2][0] = c44 * phi44;
+  Y_AlGaAs_Phi[2][0][0][2] = c44 * phi44;
+  Y_AlGaAs_Phi[2][0][2][0] = c44 * phi44;
+
+  Y_AlGaAs_Phi[1][2][1][2] = c44 * phi44;
+  Y_AlGaAs_Phi[1][2][2][1] = c44 * phi44;
+  Y_AlGaAs_Phi[2][1][1][2] = c44 * phi44;
+  Y_AlGaAs_Phi[2][1][2][1] = c44 * phi44;
+
   // Define some dimensionless loss angles.
   // lossPhi_Iso_FusedSilica = fused silica
   // lossPhi_Iso_Ta2O5 = Tantalum oxide
   // lossAlGaAsx92 = x=0.92 AlGaAs
   // lossPhi = 1.e-7; //Chalermsongsak+ (2015) Table I
-  lossPhi_Iso_FusedSilica = 1.e-6;  // Cole+ (2013)
-  lossPhi_Iso_Ta2O5 = 4.e-4;        // Yamamoto+ (2006), PRD 74, 022002
-                              // (good value??)
-  lossPhi_Iso_AlGaAs = 2.5e-5;          // Cole+ (2013)
-  lossPhi_AlGaAs = lossPhi_Iso_AlGaAs;  // use same loss angle from Cole+(2013)
+  lossPhi_Iso_FusedSilica = config["Mirror"]["LossAngles"]["Fused_Silica"]
```

Comment:
I prefer still hardcoding these, rather than having the user specify them. Ok to make a “custom” material that does have extra options. Otherwise it’s easy to screw up the parameters as a user running the code. But open an issue about this for now